portability-api-launcher/src/main/java/org/dataportabilityproject/api/launcher/TypeManager.java

```diff
@@ -13,7 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.dataportabilityproject.spi.cloud.types;
+package org.dataportabilityproject.api.launcher;
```

Comment:
We briefly talked about this in the call, should we rename portability-api-launcher? Since nothing gets "launched" from here and this isnt the entry point of the api. Also the context and extensions used are not unique to the gateway.

Reply:
Sounds good to me. Should we do this in another PR? Also, does anyone have a preference/suggestion for name?

Reply:
another PR sgtm - how about "portability-bootstrap" or maybe "portability-base" 🤔 naming things is hard.

Reply:
bootstrap sounds good